airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -264,6 +276,21 @@ def _start_task(self):
         self.log.info('ECS Task started: %s', response)
 
         self.arn = response['tasks'][0]['taskArn']
+        ecs_task_id = self.arn.split("/")[-1]
+        self.log.info(f"ECS task ID is: {ecs_task_id}")
+
+        if self.reattach:
+            # Save the task ARN in XCom to be able to reattach it if needed
+            self._xcom_set(context, key="ecs_task_arn", value=self.arn, task_id=f"{self.task_id}_task_arn")
```

Comment:
Minor nit: the magic string `"ecs_task_arn"` is used in a couple places. Perhaps put this in a constant. You could also use a templated string constant for the task_id